app/workspaces/transfers/workspace.go

```diff
@@ -10,7 +10,7 @@ import (
 
 type Workspace interface {
 	List(ctx context.Context, filter transfer.Filter) ([]Reference, error)
-	Create(ctx context.Context, req CreateInput) (*CreateOutput, error)
+	Create(ctx context.Context, req CreateInput) (CreateOutput, error)
```

Comment:
Talvez o nome do parametro poderia ser um nome mais associado ao que estamos recebendo nesse método ou na interface pode usar somente os tipos. Assim diminui a carga cognitiva. O que você acha?
 
```suggestion
	Create(ctx context.Context, createInput CreateInput) (CreateOutput, error)
```
```suggestion
	Create(context.Context, CreateInput) (CreateOutput, error)
```

Reply:
Perfeito!!! Não sabia que não precisava nomear os parâmetros em interfaces. Vou adicionar um issue para dar uma melhorada nesse ponto. Thaaaanks